viadot/tasks/duckdb.py

```diff
@@ -126,7 +126,7 @@ def run(
             NoReturn: Does not return anything.
         """
         try:
-            check_if_empty_file(path=path, if_empty=if_empty, file_extension=".parquet")
+            check_if_empty_file(path=path, if_empty=if_empty)
```

Comment:
If you default to `None` for `if_empty` here and you pass it over to `check_if_empty_file()`, it will be `None` in `check_if_empty_file()` (NOT the default value you set in `check_if_empty_file()` -- the default is only used if you don't pass the parameter at all). 

There are two ways to solve this: 
1) Set the default here to the same default as in `check_if_empty_file()`, 
2) Check for `None` in `check_if_empty_file()`, eg. `if_empty = if_empty or "warn"`.